docs/guides/testing-deployments.md

```diff
@@ -2,11 +2,16 @@
 title: Testing Deployments
 ---
 
-Using the Zuplo GitHub integration, tests can be run after a deployment and used to block pull requests from being merged. This can help ensure that changes to your Zuplo gateway won't break your production environment.
+:::tip
+
+These instructions assume that you are using custom GitHub Action workflow, in conjunction with the Zuplo deployer. If you prefer setting up your own CI/CD for more fine-grained control, please take a look at [running your own CI/CD](../guides/custom-ci-cd.md).
```

Comment:
@ntotten - I think this page still provides value until the deployer is fully integrated to also run tests on behalf of the user.